decoder/src/main/java/openlr/decoder/properties/OpenLRDecoderProperties.java

```diff
@@ -138,6 +141,7 @@ public OpenLRDecoderProperties(final Configuration config)
         dnpVariance = OpenLRPropertyAccess.getIntegerPropertyValue(config,
                 OpenLRDecoderProperty.DNP_VARIANCE);
         maxBearingDiff = OpenLRPropertyAccess.getIntegerPropertyValueFromMap(config, OpenLRDecoderProperty.BEAR_RATING, "MaxBearingDiff");
+        minBearingScore = OpenLRPropertyAccess.getIntegerPropertyValueFromMap(config, OpenLRDecoderProperty.BEAR_RATING, "MinScore");
```

Comment:
Hi David, as you mentioned via Slack: maybe add a check if the value of the minBearingScore is in correct range (0 to 1)?